src/parser.rs

```diff
@@ -125,6 +125,32 @@ impl fmt::Display for Expression {
     }
 }
 
+#[derive(PartialEq, Debug, Clone)]
+pub struct ProcedureCall {
```

Comment:
Seems unnecessary to intro a new struct. It will created non reader friendly enum variant code.

Reply:
I think new struct is necessary for simplify code [here](https://github.com/Danielmelody/Ruschm/blob/d869a091d9fae7196d8f3e71efd3147fe4ba1c77/src/interpreter/mod.rs#L322-L325) and [here](https://github.com/Danielmelody/Ruschm/blob/d869a091d9fae7196d8f3e71efd3147fe4ba1c77/src/interpreter/mod.rs#L398-L399).
Or else I would need to unpack enum variant to tuple, pass tuple around, maybe intro a new type alias, etc.